pkg/cmd/install/hubAddon/cmd.go

```diff
@@ -1,5 +1,5 @@
 // Copyright Contributors to the Open Cluster Management project
-package addons
+package hubAddon
```

Comment:
Usual go convention would be to keep this all lowercase: https://go.dev/blog/package-names